jsonnetfile.lock.json

```diff
@@ -8,7 +8,7 @@
                     "subdir": "jsonnet/kube-thanos"
                 }
             },
-            "version": "21e7b40d93c85e8c53124b7b953176649a76d5a4"
+            "version": "eea3aeff15e551300c6dd7d472c99ffbb4c0b166"
```

Comment:
/hold

not sure why this is being bumped here

Reply:
`git checkout -- jsonnetfile.lock.json` and then run `jb install`. I don't think you need to update any external deps, do you?

Reply:
```
~/Git/configuration ⑂cache-frontend* $ git pull --rebase upstream master
From github.com:Go-SIP/configuration
 * branch            master     -> FETCH_HEAD
Current branch cache-frontend is up to date.
```

Looks like I'm up-to-date with upstream/master. Still not sure what's causing the dependency versions to be bumped in the lockfile :)